Assets/Scripts/Missions/MissionCafeteria1/MissionCafeteria1.cs

```diff
@@ -108,7 +108,7 @@ private void SpawnEnemies(List<MissionEnemy> enemies)
             if (NavMesh.SamplePosition(enemy.spawnPosition, out NavMeshHit closestNavmeshHit, 10.0f, NavMesh.AllAreas))
             {
                 GameObject spawnedEnemy = Instantiate(enemy.prefab, closestNavmeshHit.position, Quaternion.Euler(enemy.spawnRotation));
-                NursePatrol patrol = Utils.GetRequiredComponent<NursePatrol>(spawnedEnemy, $"Enemy in MissionCafeteria1 does not have a NursePatrol component!");
+                Patroller patrol = Utils.GetRequiredComponent<Patroller>(spawnedEnemy, $"Enemy in MissionCafeteria1 does not have a NursePatrol component!");
```

Comment:
Update the comment here too?

Reply:
Oops yeah good catch, I just hit "refactor" without the "refactor string occurences" option on